src/store/modules/brushModule.js

```diff
@@ -136,31 +136,6 @@ function enabledElementCallback(enabledElement) {
   setters.elementVisible(enabledElement);
 }
 
-/**
- * RemoveEnabledElementCallback - Element specific memory cleanup.
- * @public
- * @param  {Object} enabledElement  The element being removed.
- * @returns {void}
- */
-// TODO -> Test this before adding it to the module.
-function removeEnabledElementCallback(enabledElement) {
-  if (!external.cornerstone) {
-    return;
-  }
-
-  const cornerstoneEnabledElement = external.cornerstone.getEnabledElement(
-    enabledElement
-  );
-
-  const enabledElementUID = cornerstoneEnabledElement.uuid;
-  const colormap = external.cornerstone.colors.getColormap(state.colorMapId);
-  const numberOfColors = colormap.getNumberOfColors();
-
-  // Remove enabledElement specific data.
-  delete state.visibleSegmentations[enabledElementUID];
-  delete state.imageBitmapCache[enabledElementUID];
-}
```

Comment:
Was this removed because there are no references/calls to it, and it's not exported?

Reply:
CC: @JamesAPetts, just a heads up that this was removed. It looks like we never exported it as part of the module.